x/bank/client/rest/sendtx.go

```diff
@@ -54,13 +54,19 @@ func SendRequestHandlerFn(cdc *codec.Codec, kb keys.Keybase, cliCtx context.CLIC
 			return
 		}
 
-		info, err := kb.Get(req.BaseReq.Name)
-		if err != nil {
-			utils.WriteErrorResponse(w, http.StatusBadRequest, err.Error())
+		if req.BaseReq.GenerateOnly {
```

Comment:
Is there any way to generalize this logic so that we can use the same function for each REST route and ensure `GenerateOnly` is respected?

Reply:
Hmmm, maybe? I'll have to implement the other routes first and see what can be DRY-ed up.

Reply:
Oh! Absolutely! I just don't want to see the same code at the end of every route and CLI command.

Reply:
Ok so because there _can_ exist custom business logic in each module's REST handlers that may require access to the local Keybase (e.g. `x/staking`), I don't think we can really consolidate it outside of maybe some helper function within each module.